### PR TITLE
Added count for number of shifts created on toast message. Closes #1158.

### LINKED ISSUE #1158: Display '4 shifts successfully created..." in toastr msg for recurring shifts to alert user they just created all those shifts

### PR DESCRIPTION
**Pull request by michaelDpierce**

### CODE REVIEW

**Comment by michaelDpierce** [comment]
Tested on staging :+1:

File: app/assets/javascripts/controllers/scheduling/shifts-modal.js.coffee [hunk 1]
```diff
@@ -80,7 +80,12 @@
         .then (newShifts) =>
           if newShifts
             @user.shifts = @user.shifts.concat newShifts
-            toastr.success 'Shift was successfully created.'
+            if newShifts.length > 1
+              numberShifts = newShifts.length.toString()
+              toast = numberShifts.concat(' Shifts were successfully created.')
```
Comment (duffcodester) [inline app/assets/javascripts/controllers/scheduling/shifts-modal.js.coffee]: lower case Shifts maybe?

**Comment by duffcodester** [comment]
:+1:

File: app/assets/javascripts/controllers/scheduling/shifts-modal.js.coffee (hunk 1 above)
Comment (michaelDpierce) [inline app/assets/javascripts/controllers/scheduling/shifts-modal.js.coffee]: We have been doing uppercase throughout the app on toasts.

**Pull request merged**